hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/cmd/FeedCommProcessor.java

```diff
@@ -185,17 +187,25 @@ public void run() {
                         String messageString = ApiDeserializer.toHawkularFormat(message);
                         Buffer buffer = new Buffer();
                         buffer.writeUtf8(messageString);
-                        FeedCommProcessor.this.webSocket.sendMessage(WebSocket.PayloadType.TEXT, buffer);
+                        RequestBody requestBody = RequestBody.create(WebSocket.TEXT, buffer.readByteArray());
+                        FeedCommProcessor.this.webSocket.sendMessage(requestBody);
                     } else {
                         BinaryData messageData = ApiDeserializer.toHawkularFormat(message,
                                 messageWithData.getBinaryData());
-                        BufferedSink sink = FeedCommProcessor.this.webSocket
-                                .newMessageSink(WebSocket.PayloadType.BINARY);
-                        try {
-                            emitToSink(messageData, sink);
-                        } finally {
-                            sink.close();
-                        }
+
+                        RequestBody requestBody = new RequestBody() {
+                            @Override
+                            public MediaType contentType() {
+                                return WebSocket.BINARY;
+                            }
+
+                            @Override
+                            public void writeTo(BufferedSink bufferedSink) throws IOException {
+                                    emitToSink(messageData, bufferedSink);
+                            }
+                        };
+
+                        FeedCommProcessor.this.webSocket.sendMessage(requestBody);
```

Comment:
1. Is there no more need to close anything? In above line 197 (when we had a sink) we needed to close it in finally block. Now I see sink goes away and we have a request body. Just making sure there is nothing we explicitly have to close here. (same for line 252 below - if close is needed here somewhere, it will be needed down there as well)

2. trivial - line 204 is indented improperly - looks like there are 4 extra spaces - checkstyle is probably going to fall on this line?